config/config.go

```diff
@@ -283,6 +283,10 @@ type Config struct {
 	OpenPolicyAgentEnvoyMetadata   string        `yaml:"open-policy-agent-envoy-metadata"`
 	OpenPolicyAgentCleanerInterval time.Duration `yaml:"open-policy-agent-cleaner-interval"`
 	OpenPolicyAgentStartupTimeout  time.Duration `yaml:"open-policy-agent-startup-timeout"`
+
+	// Passive Health Checks
+	EnablePassiveHealthCheck bool     `yaml:"enable-passive-health-check"`
+	PassiveHealthCheckParams mapFlags `yaml:"passive-health-check-params"`
```

Comment:
Maybe we can just have a single new flag
```go
	PassiveHealthCheck mapFlags `yaml:"passive-health-check"`
```
Then we do not have any defaults (proposed defaults are arbitrary) to make user to think about the configuration and enable PHC when `len(PassiveHealthCheck) > 0`

Reply:
I have thought about this, and yes, I agree it makes sense to get all values from the user. However, I still provide some example with arbitrary numbers in the `operation.md` which users can use as a baseline.

Reply:
We could also think the other way: provide only the Enable flag and hard code rest.
We could add env variables to switch all the "defaults" deep in the package and test in our dev cluster what values make sense. Thinking about NLB, they also have no input from the user and have PHC working.